draft-ietf-rtcweb-jsep.xml

```diff
@@ -1158,20 +1164,23 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           parameters are changed:</t>
          <t><list style="symbols">
             <t>Any changes to the STUN/TURN servers to use affect
-            the next gathering phase. If gathering has already occurred, this
-            will cause the next call to createOffer to generate new ICE
-            credentials, for the purpose of forcing an ICE restart and kicking
-            off a new gathering phase, in which the new servers will be used.
-            If the ICE candidate pool has a nonzero size, any existing
-            candidates will be discarded, and new candidates will be gathered
-            from the new servers.
+            the next gathering phase. If gathering has already occurred, the
+            'needs-ice-restart' bit mentioned in <xref target="sec.ice-gather-overview"/>
```

Comment:
Maybe
"If gathering is complete, any changes to the STUN/TURN servers affect the next gathering phase..."

and then at the end
"If STUN/TURN servers are changed during gathering, then no ICE restart is needed"